File: Python3/cadastro_cliente.py

"""Faça um algoritmo que leia um cadastro de cliente, o seu nome, sua idade e o seu salário.
e por fim, mostre os valores."""

class cadastro:
    nome = ""
    idade = 0
    salario = 0.0
       
dados_cliente = cadastro()
dados_cliente.nome    =     input("Digite o nome do Cliente: ")
dados_cliente.idade   = int(input("Digite a idade do Cliente: "))
dados_cliente.salario = int(input("Digite o salario do Cliente: "))
print("Nome do Cliente: ",dados_cliente.nome)
print("Idade do Cliente: ",dados_cliente.idade)
print("Salário do Cliente: ",dados_cliente.salario)      
   

File: java/cadastro_cliente.java
import java.util.Scanner;

class cadastro_cliente {
  public static void main(String args[]) {
    cadastro dados_cliente = new cadastro();
    Scanner sc = new Scanner(System.in);
    System.out.println("Digite o nome do Cliente");
    dados_cliente.nome = sc.nextLine();
    System.out.println("Digite a idade Cliente");
    dados_cliente.idade = sc.nextInt();
    System.out.println("Digite o salario do Cliente");
    dados_cliente.salario = sc.nextFloat();

    System.out.println("Nome do Cliente .: " + dados_cliente.nome + "\nIdade do Cliente .: " + dados_cliente.idade
        + "\nSalario do cliente .: " + dados_cliente.salario);

    sc.close();
  }
}

class cadastro {
  String nome;
  int idade;
  float salario;
}
